libs/s25main/desktops/dskGameInterface.cpp

```diff
@@ -1153,18 +1155,6 @@ void dskGameInterface::CI_ReplayEndReached(const std::string& msg)
 void dskGameInterface::CI_GamePaused()
 {
     messenger.AddMessage(_("SYSTEM"), COLOR_GREY, ChatDestination::System, _("Game was paused."));
-
-    /// Straßenbau ggf. abbrechen, wenn aktiviert
-    if(road.mode != RoadBuildMode::Disabled)
```

Comment:
You will not be stuck in road mode since you can still manually cancel road building.

That said, my reasoning for removing this is the same as above and if you prefer to have it, no real problem with me.

Reply:
Ah, ok, then that is fine too.

So only the braces please (in general I avoid braces if the `if` and statement below or short (single line), but here we have a comment and a statement and a very long if so the braces help with readability) -> https://github.com/Return-To-The-Roots/s25client/pull/1372#discussion_r575804883